README.md

```diff
@@ -32,42 +32,95 @@ DocSearch crawls your documentation, pushes the content to an Algolia index and
 
 > Don't have your Algolia credentials yet? [Apply to DocSearch](https://docsearch.algolia.com/apply)!
 
-**1.** Import the library as an ECMAScript module:
+### JavaScript
+
+#### Installation
 
 ```sh
-npm install @docsearch/js@alpha
-# or
 yarn add @docsearch/js@alpha
+# or
+npm install @docsearch/js@alpha
 ```
 
-```js
-import docsearch from '@docsearch/js';
+If you don’t want to use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@alpha"></script>
 ```
 
-**1–bis.** Or with a script tag (at the end of the `body`):
+#### Get started
+
+To get started, you need a [`container`](https://docsearch.algolia.com/docs/api#container) for your DocSearch component to go in. If you don’t have one already, you can insert one into your markup:
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@alpha"></script>
+<div id="docsearch"></div>
 ```
 
-**2.** Use the library:
+Then, insert DocSearch into it by calling the [`docsearch`](https://docsearch.algolia.com/docs/api) function and providing the container. It can be a [CSS selector](https://developer.mozilla.org/en-us/docs/web/css/css_selectors) or an [Element](https://developer.mozilla.org/en-us/docs/web/api/htmlelement).
+
+Make sure to provide a [`container`](https://docsearch.algolia.com/docs/api#container) (for example, a `div`), not an `input`. DocSearch generates a fully accessible search box for you.
+
+```js app.js
+import docsearch from '@docsearch/js';
+
+import '@docsearch/css';
 
-```js
 docsearch({
   container: '#docsearch',
+  appId: 'YOUR_APP_ID',
   indexName: 'YOUR_INDEX_NAME',
-  apiKey: 'YOUR_API_KEY',
+  apiKey: 'YOUR_SEARCH_API_KEY',
 });
 ```
 
-**3.** [Customize the color scheme](https://docsearch.algolia.com/docs/styling/).
+### React
+
+#### Installation
+
+```bash
+yarn add @docsearch/react@alpha
+# or
+npm install @docsearch/react@alpha
+```
+
+If you don’t want to use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/react@alpha"></script>
+```
+
+#### Get started
+
+DocSearch generates a fully accessible search box for you.
+
+```jsx App.js
+import { DocSearch } from '@docsearch/react';
+
+import '@docsearch/css';
+
+function App() {
+  return (
+    <DocSearch
+      appId="YOUR_APP_ID"
+      indexName="YOUR_INDEX_NAME"
+      apiKey="YOUR_SEARCH_API_KEY"
+    />
+  );
+}
+
+export default App;
+```
+
+## Styling
+
+[Read documentation →](https://docsearch.algolia.com/docs/styling)
 
 ## Related projects
 
 DocSearch is made of the following repositories:
 
 - **[algolia/docsearch](https://github.com/algolia/docsearch)**: DocSearch source code.
-- **[algolia/docsearch-website](https://github.com/algolia/docsearch-website)**: DocSearch website and documentation.
+- **[algolia/docsearch-website](https://github.com/algolia/docsearch/tree/next/packages/website)**: DocSearch website and documentation.
```

Comment:
Or since we refer to paths here, wdyt?

```suggestion
- **[algolia/docsearch/packages/website](https://github.com/algolia/docsearch/tree/next/packages/website)**: DocSearch website and documentation.
```